modules/backend/ServiceProvider.php

```diff
@@ -88,6 +90,14 @@ protected function registerAssetBundles()
         });
     }
 
+    /**
+     * Register database factories
+     */
+    protected function registerFactories()
+    {
+        App::make(Factory::class)->load(__DIR__ . '/factories');
```

Comment:
woah woah woah, why do we suddenly need factories?

Reply:
@LukeTowers This is used for generating the authenticated user fixture for testing the permissions. I would also be keen to use it down the line for some other unit tests as well.

Reply:
Let's get @daftspunk's input on this

Reply:
@LukeTowers As per @daftspunk's recommendations, this will be moved to the tests folder and only available for tests.

Reply:
Further to the above, factories are now completely removed and replaced with a UserFixture class.